homeassistant/components/igloohome/strings.json

```
@@ -0,0 +1,19 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Client Credentials",
+        "description": "Add a set of client credentials created under your iglooaccess account.",
+        "data": {
+          "client_id": "Client ID",
+          "client_secret": "Client Secret"
+        }
```

```suggestion
        },
        "data_description": {
             "client_id": "<Some description what this is>",
              "client_secret": "<Some description what this is>",
         }
```
please also add data descriptions